runtime/src/main/cpp/Memory.cpp

```
@@ -2953,6 +3035,20 @@ KInt Kotlin_native_internal_GC_getThreshold(KRef) {
 #endif
 }
 
+void Kotlin_native_internal_GC_setCollectCyclesThreshold(KRef, KLong value) {
+#if USE_GC
+  setGCCollectCyclesThreshold(value);
```

Type mismatch here: `KLong` gets silently truncated to `KInt`.